pkg/reconciler/brokercell/controller.go

```diff
@@ -125,3 +138,17 @@ func handleResourceUpdate(impl *controller.Impl) cache.ResourceEventHandler {
 	// will be skipped by the function.
 	return controller.HandleAll(impl.EnqueueLabelOfNamespaceScopedResource(namespaceLabel, resources.BrokerCellLabelKey))
 }
+
+// reportLatency estimates the time spent since the last update of the resource object and records it to the latency metric
+func reportLatency(ctx *context.Context, resourceObj interface{}, latencyReporter *metrics.BrokerCellLatencyReporter, resourceKind, resourceName, namespace string) {
```

Comment:
Why `*context.Context`?

Reply:
Yeah, passing by reference was an over-optimization here, given that Context is an interface. Have switched to pass-by-value.